recipes/threadpool/all/conanfile.py

```diff
@@ -0,0 +1,35 @@
+import os
+from conans import ConanFile, tools
+
+class threadpoolConan(ConanFile):
+    name = "threadpool"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/progschj/ThreadPool"
+    description = "A simple C++11 Thread Pool implementation."
+    license = "Zlib"
+    topics = ("C++11", "Thread", "Pool", "threadpool", "conan")
+    settings = "os"
```

Comment:
```suggestion
    settings = "os", "compiler"
```

Reply:
Why do we need to keep the compiler in the header-only libraries? I asked you in another PR, maybe you missed it.

Reply:
https://github.com/conan-io/conan-center-index/issues/1957